proto/Cargo.toml

```diff
@@ -29,7 +29,7 @@ serde = { version = "1.0", default-features = false, features = ["derive"] }
 serde_bytes = { version = "0.11", default-features = false, features = ["alloc"] }
 subtle-encoding = { version = "0.5", default-features = false, features = ["hex", "base64", "alloc"] }
 num-traits = { version = "0.2", default-features = false }
-num-derive = { version = "0.3", default-features = false }
+num-derive = { version = "0.4", default-features = false }
```

Comment:
I've been wondering what do we use num-traits for when `prost-build` provides the conversions from `i32` [out of the box](https://docs.rs/prost/latest/prost/#enumerations).

Reply:
https://github.com/informalsystems/tendermint-rs/pull/1389/commits/5d23beadfb2f03f80bd86981f19cffa3c3407090